Explain unclear semantics of property `$ref` in Path Item Object (#1964)
OAI/OpenAPI-Specification: versions/3.0.3.md

Field Name | Type | Description
---|:---:|---

> <a name="pathItemRef"></a>$ref | `string` | Allows for an external definition of this path item. The referenced structure MUST be in the format of a [Path Item Object](#pathItemObject). A Path Item Object containing a `$ref` property is **not** a [Reference Object](#referenceObject). The properties of the referenced structure are merged with the local Path Item Object. If the same property exists in both, the referenced structure and the local one, this is a conflict. In this case the behavior is *undefined*.

Author: ```suggestion
<a name="pathItemRef"></a>$ref | `string` | Allows for an external definition of this path item. The referenced structure MUST be in the format of a [Path Item Object](#pathItemObject). A Path Item Object containing a `$ref` property is **not** a [Reference Object](#referenceObject) but the semantics of the `$ref` property are the same. The properties of the referenced structure are merged with the local Path Item Object. If the same property exists in both, the referenced structure and the local one, this is a conflict. In this case the behavior is *undefined*.
```

Author: Hm... GitHub is unable to highlight the change in my suggestion above, here is a better diff:

```diff
-A Path Item Object containing a `$ref` property is **not** a [Reference Object](#referenceObject).
+A Path Item Object containing a `$ref` property is **not** a [Reference Object](#referenceObject) but the semantics of the `$ref` property are the same.
```

Reviewer: ```suggestion
<a name="pathItemRef"></a>$ref | `string` | Allows for an external definition of this path item. The referenced structure MUST be in the format of a [Path Item Object](#pathItemObject).  In case a Path Item Object field appears both in the defined object and the referenced object, the behavior is undefined.
```

Adding the suggestion based on the discussion so we can merge the PR and release 3.0.3.
